docs/src/modules/brandingTheme.ts

```diff
@@ -436,7 +436,7 @@ export function getThemedComponents(): ThemeOptions {
         styleOverrides: {
           root: ({ theme, ownerState }) => ({
             ...(ownerState.size === 'large' && {
-              padding: theme.spacing(1.5, 1.25, 1.5, 1.5), // 12px 10px
+              padding: theme.spacing('12px', '11px', '12px', '14px'),
```

Comment:
```suggestion
              padding: theme.spacing('12px', '12px', '12px', '14px'),
```

Any reason to not round that 11 in 12? It looks good still, to me, at least!

Reply:
It was at 10px before, 12 sounds good